lib/components/AlbumScreen/track_list_tile.dart

```diff
@@ -229,15 +270,16 @@ class TrackListTile extends StatelessWidget {
             mainAxisSize: MainAxisSize.min,
             children: [
               Icon(
-                TablerIcons.playlist,
+                (currentSettings.itemSwipeActionLeftToRight == ItemSwipeActions.addToQueue) 
+                    ? TablerIcons.playlist
+                    : (currentSettings.itemSwipeActionLeftToRight == ItemSwipeActions.addToNextUp)
+                        ? TablerIcons.corner_right_down_double
+                        : TablerIcons.corner_right_down,
```

Comment:
But for *this*, you could add a method here that returns the proper icon ^^  
Alternatively, there's an inline version of the switch block that you could use.